2 Presentations - February 16th 2021

raytobing opened the pull request.

raytobing committed:
Ray's Ballpark Assignment 1

llorracc commented:
Ray,

I just realized that I missed our appointment today at 2pm.

I apologize.

In the future, I would urge you to send me an email notification of the
appointment, so that I can be sure to put a reminder on my calendar.  (I'd
be grateful if you could check whether the "rules" I have posted urge
students to send me a notification; if not, I definitely need to add that
as a "rule").



On Mon, Oct 5, 2020 at 9:34 PM Ray Tobing <notifications@github.com> wrote:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/econ-ark/ballpark/pull/19
> Commit Summary
>
>    - Ray's Ballpark Assignment 1
>
> File Changes
>
>    - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/Figure4.png
>    <https://github.com/econ-ark/ballpark/pull/19/files#diff-5e10a8854b1bd72d35648b5e0f8fed67>
>    (0)
>    - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/Figure5.png
>    <https://github.com/econ-ark/ballpark/pull/19/files#diff-c1accbada570f40b204abc2b63f98866>
>    (0)
>    - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/MK3TagTax - Syareza
>    Tobing.ipynb
>    <https://github.com/econ-ark/ballpark/pull/19/files#diff-045039bb7f75d61de8fa25a9baa1de94>
>    (196)
>    - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/MK3TagTax - Syareza
>    Tobing.pdf
>    <https://github.com/econ-ark/ballpark/pull/19/files#diff-8cd0f74fd73ab719515df4a841fea53a>
>    (0)
>    - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/Table5.png
>    <https://github.com/econ-ark/ballpark/pull/19/files#diff-90d7c6972d8551a919dd99f7831ab4aa>
>    (0)
>    - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/Table6.png
>    <https://github.com/econ-ark/ballpark/pull/19/files#diff-07dea625743a720934e7d4dcc20efd95>
>    (0)
>
> Patch Links:
>
>    - https://github.com/econ-ark/ballpark/pull/19.patch
>    - https://github.com/econ-ark/ballpark/pull/19.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/econ-ark/ballpark/pull/19>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKCK7ZPIYA7Y7GYZPS6A73SJJXZ7ANCNFSM4SFMMKOA>
> .
>


-- 
- Chris Carroll

raytobing commented:
Good evening Professor.

No worries. I'll try and make another appointment for next week and notify you through your e-mail a day before.

I've also checked your rules and there are no mentions regarding sending you a notification.

Thank you and see you tomorrow.



Sincerely,

Syareza Tobing.
________________________________
From: Christopher Llorracc Carroll <notifications@github.com>
Sent: Monday, October 5, 2020 10:29 PM
To: econ-ark/ballpark
Cc: Syareza Tobing; Author
Subject: Re: [econ-ark/ballpark] Ray's Ballpark Assignment 1 (#19)


      External Email - Use Caution



Ray,

I just realized that I missed our appointment today at 2pm.

I apologize.

In the future, I would urge you to send me an email notification of the
appointment, so that I can be sure to put a reminder on my calendar. (I'd
be grateful if you could check whether the "rules" I have posted urge
students to send me a notification; if not, I definitely need to add that
as a "rule").



On Mon, Oct 5, 2020 at 9:34 PM Ray Tobing <notifications@github.com> wrote:

> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/econ-ark/ballpark/pull/19
> Commit Summary
>
> - Ray's Ballpark Assignment 1
>
> File Changes
>
> - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/Figure4.png
> <https://github.com/econ-ark/ballpark/pull/19/files#diff-5e10a8854b1bd72d35648b5e0f8fed67>
> (0)
> - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/Figure5.png
> <https://github.com/econ-ark/ballpark/pull/19/files#diff-c1accbada570f40b204abc2b63f98866>
> (0)
> - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/MK3TagTax - Syareza
> Tobing.ipynb
> <https://github.com/econ-ark/ballpark/pull/19/files#diff-045039bb7f75d61de8fa25a9baa1de94>
> (196)
> - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/MK3TagTax - Syareza
> Tobing.pdf
> <https://github.com/econ-ark/ballpark/pull/19/files#diff-8cd0f74fd73ab719515df4a841fea53a>
> (0)
> - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/Table5.png
> <https://github.com/econ-ark/ballpark/pull/19/files#diff-90d7c6972d8551a919dd99f7831ab4aa>
> (0)
> - *A* models/We-Would-Like-In-Econ-ARK/MK3TagTax/Table6.png
> <https://github.com/econ-ark/ballpark/pull/19/files#diff-07dea625743a720934e7d4dcc20efd95>
> (0)
>
> Patch Links:
>
> - https://github.com/econ-ark/ballpark/pull/19.patch
> - https://github.com/econ-ark/ballpark/pull/19.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/econ-ark/ballpark/pull/19>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKCK7ZPIYA7Y7GYZPS6A73SJJXZ7ANCNFSM4SFMMKOA>
> .
>


--
- Chris Carroll

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fecon-ark%2Fballpark%2Fpull%2F19%23issuecomment-703990802&data=02%7C01%7Cmtobing1%40jhu.edu%7C47c3b84043b0489bd56008d8699f65e9%7C9fa4f438b1e6473b803f86f8aedf0dec%7C0%7C0%7C637375481987305779&sdata=cZSTZzeNZZVY5yQvsN7FG8%2BBEy2kqWrNlIn8Ce55h%2F0%3D&reserved=0>, or unsubscribe<https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAQ3G57WOOGDPKWXPAX3L6BLSJJ6BVANCNFSM4SFMMKOA&data=02%7C01%7Cmtobing1%40jhu.edu%7C47c3b84043b0489bd56008d8699f65e9%7C9fa4f438b1e6473b803f86f8aedf0dec%7C0%7C0%7C637375481987315768&sdata=yzh316CirdX0C5BfNq15bKkCcyBKiA4W1UZ9k0hHBxk%3D&reserved=0>.

llorracc commented:
Clean up math a bit; fix typos.

raytobing committed:
Delete MK3TagTax - Syareza Tobing.ipynb

raytobing committed:
Delete MK3TagTax - Syareza Tobing.pdf

raytobing committed:
Fixed typos 

I couldn't fix the math errors as it appeared on your Jupyter Notebook because the same error did not appear on mine.

raytobing committed:
Ballpark_2_Assignment

raytobing commented:
Ray's second Ballpark Assignment using Buffer Stock Theory Template

raytobing committed:
Delete econtexRoot.tex

raytobing committed:
cleanup

raytobing committed:
fixed typo

raytobing committed:
Modified folder with all necessary folders

raytobing committed:
Updated overview

raytobing committed:
Ray's slides assignment

raytobing committed:
Fixed graphs on slides

raytobing committed:
2 new presentations

raytobing committed:
Presentation February 16th 2021
